athena/tests/modules/text/utils/mock_llm_config.py

```
@@ -0,0 +1,22 @@
+
+def mock_get_llm_config(*args, **kwargs):
+
+    mock_config = Mock()
+    mock_config.models = Mock()
+    mock_config.models.base_model_config = Mock()
+    mock_config.models.mini_model_config = Mock()
+    mock_config.models.fast_reasoning_model_config = Mock()
+    mock_config.models.long_reasoning_model_config = Mock()
+    
+    for model_config in [mock_config.models.base_model_config, 
+                        mock_config.models.mini_model_config,
```

Flake8 (7.2.0)

[error] continuation line under-indented for visual indent

(E128)